lib/battle_boats/console_ui.rb

```diff
@@ -35,59 +58,26 @@ def win
 
     private
 
-    attr_reader :output, :input
+    attr_reader :output, :input, :board_formatter
 
-    def valid_input?(coordinate)
-      coordinate =~ /^[A-J][0-9]$/i
+    def valid_orientation_input?(orientation)
+      orientation =~ /^[h{1}|v{1}]$/i
     end
 
     def input_to_coordinate(input)
       input_row = input[0]
```

Comment:
Line [34](https://github.com/Thomascountz/battle_boats/pull/17/files#diff-c6d72b6b0746378f6e51ed1d6a42e90dR34), `BoardFormatter` is responsible for validating the input, but here, `ConsoleUI` is taking the responsibility of turning that input into a `Coordinate`, and is intimately dependent on details of the input's format.

Is there a way that the details of the `user_input`'s format could live in only one place? If we changed the valid format of a user's input, where would we expect to look to make that change? Would we expect to have to look in more than one place?